### DISCUSSION TITLE
How to set pre selected value

### DISCUSSION THREAD

**Discussion by codingmachine16** [GrapesJS/grapesjs#3719] [Q&A]
Hi @artf 
  
Here is a select box 

label: "Data",
              name: "productsDataSource",
              id: "productsDataSource",
              type: "select",
              options: [
                {
                  id: "new",
                  name: "New",
                },
                {
                  id: "sale",
                  name: "Sale",
                },
                {
                  id: "featured",
                  name: "Featured",
                },
                {
                  id: "categories",
                  name: "Categories",
                },
              ],
              changeProp: 1,
            },

how to pass pre select value to it like "please select an option"

Thanks

**Comment by mosh-tudor** [comment]
@codingmachine16 
Use `value: ''`:

```js
options: [
  {
    value: '',
    name: "Please select an option",
  },
  {
    name: "New",
  },
  {
    name: "Sale",
  },
  {
    name: "Featured",
  },
  {
    name: "Categories",
  },
]
```

**Reply by codingmachine16** [reply]
How can we have one the above value as selected ? This is the main issue
